src/main/msp/msp.c

```diff
@@ -197,6 +197,27 @@ static uint32_t getFeatureMask(void)
     }
 }
 
+static int mspDescriptor = 0;
+
+mspDescriptor_t mspDescriptorAlloc(void)
+{
+    return (mspDescriptor_t)mspDescriptor++;
+}
+
+static uint32_t mspArmingDisableFlags = 0;
+
+static void mspArmingDisableByDescriptor(mspDescriptor_t desc)
+{
+    mspArmingDisableFlags |= (1 << desc);
+}
+
+static bool mspArmingEnableByDescriptor(mspDescriptor_t desc)
```

Comment:
The return value for this seems to be a bit unintuitive - how about adding a separate function to check the arming disabled flag set status?

Reply:
Yeah, I was thinking of the same thing.

Reply:
So are you going to change this now? :wink: